docs/topics/get-involved.adoc

```diff
@@ -3,6 +3,7 @@
 // * docs/cli-guide/master.adoc
 // * docs/maven-guide/master.adoc
 
+:_content-type: CONCEPT

```

Comment:
@stoobie @sbeskin-redhat I think this is a procedure, even though it's not written like one. Maybe it should be. It's not a usage procedure, but it is something we'd like the user to know how to do.

Reply:
@RichardHoch  I don't think it can be called a procedure. It's just general recommendations.